provider/k8s/template/app/service.yml.tmpl

```diff
@@ -19,9 +19,13 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   annotations:
-    {{ range keyValue .Annotations }}
-    {{.Key}}: "{{ quoteEscape .Value}}"
-    {{ end }}
+    {{- range $key, $value := .Annotations }}
+        {{ $key }}
+        {{- $lines := splitIntoLines $value }}
+        {{- range $lines }}
+          {{ . }}
+        {{- end }}
+    {{- end }}
```

Comment:
why this change is needed?

Reply:
This is for the above said change of printing slice of strings for a annotation instead of interface.